src/components/Auth/Login/Login.react.js

```diff
@@ -13,8 +13,15 @@ import UserPreferencesStore from '../../../stores/UserPreferencesStore';
 import CustomServer from '../../ChatApp/CustomServer.react';
 import Translate from '../../Translate/Translate.react';
 import Dialog from 'material-ui/Dialog';
+import CommunicationEmail from 'material-ui/svg-icons/communication/email';
+import ActionLock from 'material-ui/svg-icons/action/lock'
+
 const cookies = new Cookies();
 
+const iconStyles = {
+  marginRight: 10,
```

Comment:
Do you think this will not affect the responsiveness?

Reply:
It doesn't. Without the `margin-right`, the icon remains on the same line as the input box if the screen size is small. See this : 

![With/Without Margin](https://i.imgur.com/P67TiUc.png)